### DISCUSSION TITLE
Disable Organization Owners' Ability to Set Account Recovery Policy in Vaultwarden

### DISCUSSION THREAD

**Discussion by ACloud22** [dani-garcia/vaultwarden#5151] [Q&A] [answered]
Hello,

I would like to know if there is a way to prevent organization owners in Vaultwarden from setting certain policies. Specifically, I would like users to retain the ability to create organizations, but I want the "Account Recovery administration" policy to be either disabled or hidden entirely for all organization owners.

![VW](https://github.com/user-attachments/assets/adc8b65e-eb76-4424-b03f-409b143b6f14)

Is this configuration possible in Vaultwarden? If so, could you guide me through the steps to achieve it?

Thank you for your assistance!

**Comment by BlackDex** [comment, marked as answer]
No, it is not. You need to enable this feature per organization, and you need to do this via the web-vault.
If this was disabled to be possible on the server side you can't enable it at all any more rather or not an org would like that.

There is an open PR which would make it possible to hide this feature via CSS. But that will only hide it, not make it impossible to activate or use it.
